website/source/docs/platform/k8s/helm.html.md

```diff
@@ -1,107 +1,17 @@
 ---
 layout: "docs"
-page_title: "Helm - Kubernetes"
+page_title: "Helm Chart Reference - Kubernetes"
 sidebar_current: "docs-platform-k8s-helm"
 description: |-
-  The Consul Helm chart is the recommended way to install and configure Consul on Kubernetes. In addition to running Consul itself, the Helm chart is the primary method for installing and configuring Consul integrations with Kubernetes such as catalog syncing, Connect injection, and more.
+  Reference for the Consul Helm chart.
 ---
 
-# Helm Chart
-
-The [Consul Helm chart](https://github.com/hashicorp/consul-helm)
-is the recommended way to install and configure Consul on Kubernetes.
-In addition to running Consul itself, the Helm chart is the primary
-method for installing and configuring Consul integrations with
-Kubernetes such as catalog syncing, Connect injection, and more.
-
-A step-by-step beginner tutorial and accompanying video can be found at the [Minikube with Consul guide](https://learn.hashicorp.com/consul/getting-started-k8s/minikube?utm_source=consul.io&utm_medium=docs).
-
-This page assumes general knowledge of [Helm](https://helm.sh/) and
-how to use it. Using Helm to install Consul will require that Helm is
-properly installed and configured with your Kubernetes cluster.
-
--> **Important:** The Helm chart is new and
-may still change significantly over time. Please always run Helm with
-`--dry-run` before any install or upgrade to verify changes.
-
-~> **Security Warning:** By default, the chart will install an insecure
-configuration of Consul. This provides a less complicated out-of-box experience
-for new users, but is not appropriate for a production setup. Make sure that
-your Kubernetes cluster is properly secured to prevent unwanted access to
-Consul, or that you understand and enable the
-[recommended Consul security features](/docs/internals/security.html).
-Currently, some of these features are not supported in the Helm chart and
-require additional manual configuration.
```

Comment:
We should move this security warning to the Running Consul page.

Reply:
It's there already on line 38

Reply:
Ah great, missed that and forgot to check the original page. Thanks!